public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json

```diff
@@ -1042,9 +1005,7 @@
     "hidden": false,
     "refresh_intervals": ["30m", "1h", "12h", "24h", "3d", "7d", "30d"]
   },
-  "timezone": "",
-  "title": "Azure Alert Consumption",
+  "title": "Azure / Alert Consumption",
```

Comment:
I'd keep the naming consistent, so you either leave it as it was or do Azure - Alert Consumption, to match the rest:

![Screenshot from 2022-06-14 11-02-35](https://user-images.githubusercontent.com/4025665/173538712-71855814-fbdf-4f07-a880-c2083e2c4308.png)

Reply:
Fixed this